src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -118,24 +117,46 @@ public void Shutdown()
         protected virtual void Dispose(bool disposing)
         {
             // Cleanup
-            this.viewLoadedParamsReference.RequestOpenDocumentationLink -= HandleRequestOpenDocumentationLink;
-            this.ViewModel.MessageLogged -= OnViewModelMessageLogged;
-            documentationBrowserMenuItem.Checked -= MenuItemCheckHandler;
-            documentationBrowserMenuItem.Unchecked -= MenuItemUnCheckedHandler;
+            if (this.viewLoadedParamsReference != null)
```

Comment:
thanks for adding all this - you might want to use the new null coalescing c# stuff - I think you can now do:

`this.viewLoadedParamsReference?.RequestOpenDocumentationLink -= HandleRequestOpenDocumentationLink` for example.

Reply:
As far as I know, the null conditional operator can not be used for assignments so it will not compile in this case.